circle.yml

```diff
@@ -10,6 +10,9 @@ dependencies:
         - sudo make dev
 test:
     override:
-        - busted -o spec/busted-print.lua --coverage spec/
+        - busted -v -o spec/busted-print.lua --coverage :
+            parallel: true
+            files:
+              - spec/**/*_spec.lua
```

Comment:
yes. If you don't it runs once and displays the result in all four containers.

Check [this one](https://circleci.com/gh/Mashape/kong/914) the busted command at the top gets explicit files to test. CircleCI build 1 image, copies it to 4 and splits the test files over the 4 images (appending the filenames to the busted command)
.